doc/getting-started.md

```diff
@@ -21,59 +19,72 @@ deployment "istio-mixer" created
 ## Connecting microservices with Istio
 
 This guide shows how to set up Istio and manipulate the proxy mesh to achieve useful behavior.
-In this example we use two microservices:
+In this example we use *CouchDB* and *NGINX* as and demonstrate how to manipulate them:
+
+* CouchDB is the official CouchDB Docker image
+* NGINX is the offical NGINX Docker image.  We will configure it to redirect all calls to CouchDB
 
-* microservice "hello" returns a constant bit of JSON.
-* microservice "frontend" calls microservice "hello"
+First, let's start CouchDB and have it join the Istio service mesh
 
-The communicating microservices are examples from Kubernetes'
-[Connecting a Front End to a Back End Using a Service](https://kubernetes.io/docs/tutorials/connecting-apps/connecting-frontend-backend/) tutorial.
+```
+# Start the official Docker CouchDB image the container in a Kubernetes Pod.  5984 is the CouchDB port.
+kubectl run couchdb --image=docker.io/library/couchdb --port 5984
+```
 
-First we will start the microservices using the Istio pattern of a
-proxy within frontend's pod. The `istioctl kube-inject` command
-injects the istio runtime proxy into kubernetes resource files. It is
-documented [here](istioctl.md#kube-inject).
+The _kubectl get deployment_ subcommand simply creates a machine readable description of a service.  The _istioctl kube-inject_ subcommand creates a machine readable description of a service including Istio components.  By piping the commands together and sending the output to _kubectl apply_ we add Istio capabilities to our service.
 
```

Comment:
The tutorial would flow better if you structure it as (a) Getting started with pre-deployed app - describe how to get startedwith an app that is fully deployed (without istio) and then showed how one could install istio, show case its capabilities, etc., (b) getting started with empty K8S cluster.

The rationale being that if I am just creating my app bit by bit, then I wouldn't have to go through the pains of kube-inject. I would rather change all my YAML templates, copy paste the istio proxy stuff and get started.